static/styles/settings.css

```diff
@@ -366,6 +366,22 @@ td .button {
     #id_realm_enable_spectator_access_label a {
         color: hsl(0, 0%, 20%);
     }
+
+    select {
+        height: 30px;
+        width: 220px;
+        padding: 4px 6px;
+        color: hsl(0, 0%, 33%);
+        border-radius: 4px;
+        border: 1px solid hsl(0, 0%, 80%);
+        cursor: pointer;
+        background-color: hsl(0, 0%, 100%);
+
+        &:disabled {
+            cursor: not-allowed;
+            background-color: hsl(0, 0%, 93%);
+        }
+    }
```

Comment:
Same comment here -- I think we want to introduce a `settings_select` CSS class. (I could even imagine it being a Handlebars template, like `settings_checkbox`, but that seems like a more complex refactor).

The other change I'd like to make as a follow-up is to get rid of the `220px` width default for select elements. We already override it with random other widths for i18n reasons in almost all of the select elements, and it's possibly the case that just changing them all to some sort of automatic width thing (or a `max-width`) would look at least as good visually regardless. So I'd rather we rethink that.

The other details all seem basically fine to apply to all our settings select elements.